datastore/blob/azure_storage_api/__init__.py

```diff
@@ -270,7 +271,7 @@ async def get_image_count(container_client, folder_name):
             blob_list = container_client.list_blobs()
             count = 0
             for blob in blob_list:
-                if (blob.name.split("/")[0] == folder_uuid) and (
+                if (blob.name.split("/")[0] == folder_name) and (
                     blob.name.split(".")[-1] == "png"
                 ):
                     count += 1
```

Comment:
How does this work with the different image formats? jpg, tif, tiff . do we save them all as png?